docs/architecture/adr-036-arbitrary-signature.md

```diff
@@ -0,0 +1,60 @@
+# ADR 036: Arbitrary Message Signature
+
+## Changelog
+
+- 28/10/2020 - Initial draft
+
+## Authors
```

Comment:
This should reference some original work you can find [here](https://github.com/cosmos/ics/pull/33).

Reply:
addressed